### PR TITLE
Initial commit of tribute page

### PR DESCRIPTION
**Pull request by SFarida**
This is a tribute page of a doctor and the main files are:
- HTML file with the main content 
- CSS file for styles

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# Hello-Microverse-2
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```
Comment (SFarida) [inline README.md:5]: Please follow this [template](https://github.com/microverseinc/readme-template) for your readme file to make it more professional

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+#image {
+    display: block;
+    max-width: 30%;
+    height: auto;
+    position: absolute;
+}
+
+#img-caption {
+  position: absolute;
+}
```
Comment (SFarida) [inline css/style.css:21]: For readability please make sure to use two spaces for indentation. 
Do not use the tab space or if your code editor allows you to edit the number of spaces the tab gives, then that will be fine.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script  src="js/index.js"></script>
+  </body>
+</html>
```
Comment (SFarida) [inline index.html:39]: Same here, please fix the indentation

**Review by SFarida — COMMENTED** [review]
You did a good job so far :grin:
However, they are some corrections to made to make it complete. Check my comments below

Required changes :recycle:
Make sure to ignore all unnecessary files by adding them to .gitignore

File: index.html (hunk 3 above)
Comment (binhussen) [reply index.html:39]: - [Optional ] please be polite and clear about the inline comment you give feedback on, and explain why it required changing aspects. 👍

File: README.md (hunk 1 above)
Comment (binhussen) [reply README.md:5]: - [Optional ] The review is polite and clear as well as neutral. 👍 👍

File: css/style.css (hunk 2 above)
Comment (binhussen) [reply css/style.css:21]: - [Optional ] please be polite 👍

**Review by binhussen — APPROVED** [review]
Hi @SFarida, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions
- [Optional ] kindly be polite and clear about the comment.
- [Optional ] please be  reasonable for you comments. 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**